packages/bahmni-widgets/src/tasks/views/ViewFormData/__tests__/ViewFormData.test.tsx

```
@@ -0,0 +1,667 @@
+          onClose={mockOnClose}
+        />,
+        { wrapper: createWrapper() },
+      );
+
+      await waitFor(() => {
+        expect(screen.getByText('NO_OBSERVATIONS_FOR_TASK')).toBeInTheDocument();
+      });
+    });
+
+    it('should display NO_OBSERVATIONS_FOR_TASK when encounterGroups is empty', async () => {
+      mockGetPatientObservationsBundle.mockResolvedValue(mockObservationAndEncounterBundle);
```

ESLint

[error] Replace `mockObservationAndEncounterBundle` with `⏎········mockObservationAndEncounterBundle,⏎······`

(prettier/prettier)